Snakefile

```diff
@@ -453,19 +453,19 @@ rule load_moa:
 
 
 # input isn't actually needed but it is a file-type loader, so a dummy file must be supplied
-rule download_pubmed_source:
-    output: f'{DATA_DIR}/local/pubmed_source.json'
+rule download_sources:
+    output: f'{DATA_DIR}/local/sources.json'
     shell: dedent(f'''\
         cd {DATA_DIR}/local
-        touch pubmed_source.json
+        touch sources.json
```

Comment:
I thought the idea was to use the src/sources.js file...

Reply:
Related: I created a new ticket yesterday for source "tydying": https://www.bcgsc.ca/jira/browse/KBDEV-1150.
The new source loader should then be run first prior to all other loaders.

Reply:
It could be run as part of 'local', maybe.